app/templates/components/user-table.hbs

```diff
@@ -1,22 +1,37 @@
-{{nav-bars/info-navbar  title="Past Orders"}}
+{{nav-bars/info-navbar  title=""}}
```

Comment:
if there is no title why are you passing an empty string? your component should take care of what happens when there is no title.